File: HackerRank Solutions/10 Days of Statistics/Day 1 Standard Deviation.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <bits/stdc++.h>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int n;
    cin >> n;
    int arr[n];
    double sum = 0;
    for(int i=0; i<n; i++){
        cin >> arr[i];
        sum += arr[i];
    }
    double mean = sum / n;
    double totalSum = 0;
    for(int i=0; i<n; i++){
        totalSum += pow(arr[i] - mean, 2);
    }
    cout << fixed << setprecision(1) << sqrt(totalSum / n) << endl;
    return 0;
}

File: HackerRank Solutions/10 Days of Statistics/Day 1 Standard Deviation/Day 1 Standard Deviation.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        int [] arr = new int[n];
        double sum = 0;
        for (int i=0; i<n; i++){
            arr[i] = scan.nextInt();
            sum += arr[i];
        }
        double mean = sum / n;
        double totalSum = 0;
        for (int i=0; i<n; i++){
            totalSum += Math.pow(arr[i] - mean, 2);
        }
        System.out.format("%.1f", Math.sqrt(totalSum / n));
    }
}